pyquil/quilatom.py

```diff
@@ -18,8 +18,6 @@
 from warnings import warn
 from fractions import Fraction
 
-from pyquil.slot import Slot
```

Comment:
Was `Slot` used for those old parametric programs?

Reply:
ya